samples/boards/st/power_mgmt/blinky/prj.conf

```
@@ -4,3 +4,6 @@ CONFIG_PM_DEVICE_RUNTIME=y
 CONFIG_PM_DEVICE_SYSTEM_MANAGED=y
 CONFIG_ASSERT=y
 CONFIG_GPIO=y
+
+# un-comment to debug but prevent system to enter the low-power modes
+#CONFIG_DEBUG=y
```

IMO we should enable the STM32-specific option here - users may want to build a Release image but still be able to debug it, so it's best to demonstrate this switch can be modified independently from `CONFIG_DEBUG`.
```suggestion
#CONFIG_STM32_ENABLE_DEBUG_SLEEP_STOP=y
```

---

@erwango, do we go with CONFIG_STM32_ENABLE_DEBUG_SLEEP_STOP=y  ?